benchmarks/utils/gen_md_report.py

```diff
@@ -0,0 +1,76 @@
+import argparse
+import datetime
+import glob
+from mdutils.mdutils import MdUtils
+import os
+import pandas as pd
+
+from collections import defaultdict
+
+def iterate_subdir(input_dir, output, hw, ts_version):
```

Comment:
why is the name iterate_subdir? This seems to be creating the md file and not just iterating over data